nni/tools/nnictl/nnictl_utils.py

```diff
@@ -186,7 +184,7 @@ def get_experiment_port(args):
         exit(1)
     experiment_config = Experiments()
     experiment_dict = experiment_config.get_all_experiments()
-    return experiment_dict[experiment_id]['port']
+    return experiment_dict[experiment_id].get('port', 'N/A')
```

Comment:
just curious, why some `get('port'` has default value 'N/A' while some others do not have default value?

Reply:
it is for print `EXPERIMENT_DETAIL_FORMAT`, if `'port'` is `None`, print `N/A`. And this place actually not need default value, I will del these default values.